packages/angular_devkit/build_angular/test/tslint/works2_spec_large.ts

```diff
@@ -9,13 +9,19 @@
 import { WorkspaceNodeModulesArchitectHost } from '@angular-devkit/architect/node';
```

Comment:
@hansl kindly note that I had to do these changes to make the tests run on windows.